### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards - ExpenseReport]: Talkback does not announce the role of 'Show History' and 'Hide History' present under 'ExpenseReport'.

### ISSUE THREAD

**Issue by Bhaskar301101** [microsoft/AdaptiveCards-Mobile#10]
### Target Platforms

Android

### SDK Version

 2023.03.31.1

### Application Name

Adaptive Card

### Problem Description

This bug is also found in FC-Jan22. Linking WI
Test Environment:
OS: Android 12
Device: S21+
Screen reader: Talkback
AdaptiveCard Android App version:  2023.03.31.1
-------
​Test Environment:​
Device: Samsung Galaxy S9
Android Version: 10
Application: Adaptive Cards Android Visualizer
Application Version: 2021.09.03.1 (38680868)
Tools: Logitech Bluetooth Keyboard
Screen Reader: Talkback

Repro Steps:

1. Launch the Adaptive cards app on Android.
2. Turn on Talkback.
3. Navigate to 'Import' control and select it.
4. Navigate to 'Sample cards' and select it.
5. Navigate to available dropdown under 'Sample cards'.
6. Navigate to available dropdown under 'Sample cards' and select it.
7. Navigate all the controls under dropdown and select 'ExpenseReport'.
8. Navigate to 'Show History' and activate it.
9. Observe the issue.

Actual Result:
Talkback does not announce the role of 'Show History' and 'Hide History' present under 'ExpenseReport'. It just announces "Show History".

Expected Result:
Talkback should announce the role of 'Show History' and 'Hide History' present under 'ExpenseReport'. It should announce as "Show History button".

User Experience:
Screen Reader users will be confused if Talkback does not announce the role of 'Show History' and 'Hide History' present under 'ExpenseReport'.

WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fforms.office.com%2fpages%2fresponsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-raians%40microsoft.com%7c6a2dded50c1c45e12a5608d9798bb984%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637674463301170115%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=3StFqNyebFAcprbeS8ca9BgKUtOHDRu8WJIr5iVU%2fXc%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/89897257/232743078-5c1c3fb7-fecf-470b-ace8-74b38122512c.mp4

### Card JSON

```JSON
NA
```

**Comment by ardlank** [comment]
The fix was introduced in version 1.5 of adaptive cards. Cards will need to use the tooltip feature in their JSONs to specify what should be read. Closing ticket

https://adaptivecards.io/blog/2021/Announcing-1.5/
https://github.com/microsoft/AdaptiveCards/issues/4668

CC: @VikrantSingh01 @karthikgopal36

**Comment by ardlank** [comment]
The fix was introduced in version 1.5 of adaptive cards. Cards will need to use the tooltip feature in their JSONs to specify what should be read. Closing ticket

https://adaptivecards.io/blog/2021/Announcing-1.5/
https://github.com/microsoft/AdaptiveCards/issues/4668

CC: @VikrantSingh01 @karthikgopal36

**Comment by vagpt** [comment]
Now, this issue is being tracked with URL below hence, adding A11yTTValidated tag.

https://github.com/microsoft/AdaptiveCards-Mobile/issues/98
